commit 7425de15476df6f7452113f98adc5dcb1ca2d36c
Author: Otto Kekäläinen <otto@debian.org>
Date:   Mon Oct 15 11:33:07 2018 +0300

    Drop usage of _SYSTEMCTL_SKIP_REDIRECT from init script (Closes: #910471)

--- a/garb/files/garb.sh
+++ b/garb/files/garb.sh
@@ -21,10 +21,6 @@
 #                    extra node to arbitrate split brain situations.
 ### END INIT INFO
 
-# On Debian Jessie, avoid redirecting calls to this script to 'systemctl start'
-
-_SYSTEMCTL_SKIP_REDIRECT=true
-
 # Source function library.
 if [ -f /etc/redhat-release ]; then
 	. /etc/init.d/functions
@@ -179,4 +175,3 @@ case "$1" in
 	echo $"Usage: $0 {start|stop|status|restart|reload}"
 	exit 2
 esac
-
